buy-sell/src/components/Login/Login.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import "./Login.css";
+
+function Login() {
+  const [errorMessages, setErrorMessages] = useState({});
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  const errorMessage = "Please enter valid credentials";
+
+  const renderErrorMessage = (name) =>
+    name === errorMessages.name && (
+      <div className="error">{errorMessages.message}</div>
+    );
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    var { username, password } = document.forms[0];
+    var userData = localStorage.getItem(username.value);
+    if (userData) {
+      userData = JSON.parse(userData);
+      if (userData.password !== password.value) {
+        setErrorMessages({ name: "password", message: errorMessage });
+      }
+      if (userData.role !== "buyer")
+        setErrorMessages({ name: "username", message: errorMessage });
+      else setIsSubmitted(true);
+    } else {
+      setErrorMessages({ name: "username", message: errorMessage });
+      console.log("in this");
+    }
+  }
+
+  const renderForm = (
+    <div className="form">
+      <form onSubmit={handleSubmit}>
+        <div className="input-container">
+          <label>Username </label>
+          <input type="text" name="username" required />
+          {renderErrorMessage("username")} 
+        </div>
+        <div className="input-container">
+          <label>Password </label>
+          <input type="password" name="password" required />
+          {renderErrorMessage("password")}
+        </div>
+        <div className="button-container">
+          <input type="submit" />
+        </div>
+      </form>
+    </div>
+  );
+
+  return (
+    <div className="app">
+      <div className="login-form">
+        <div className="title">Sign In</div>
+        {isSubmitted ? <div>User is successfully logged in</div> : renderForm}
```

Comment:
Currently, Inside the Login Form we are getting "user is successfully loggedIn" with isSubmitted true.
Few Feedbacks : 
1. Maybe you can rename the flag IsSubmitted to IsloggedIn, that conveys better that the user is logged in.

2. HTML structure a bit misleading when use is logged in, that div.login-form contains not form but User is logged in successful message.